fastsnmpy.py

```diff
@@ -30,7 +30,7 @@
 
 import netsnmp
 from multiprocessing import Queue, Pool
-import json
```

Comment:
Probably better to not rename to json... will cause confusion while looking at the code.